### ISSUE TITLE
Stores doesn't consume goods from their internal storage

### ISSUE THREAD

**Issue by Shaggy84675** [voxeltycoon/bug-tracker#125]
**Describe the bug**
Stores suddenly stopped consuming goods from their local storage and started shrinking. Some of them also shows negative amount of delivered goods in last 30 days.

**Screenshots**
![unknown](https://user-images.githubusercontent.com/1374803/61865023-c9971e00-aed2-11e9-9293-186294b04e36.png)
![unknown](https://user-images.githubusercontent.com/1374803/61865075-de73b180-aed2-11e9-8cd9-41e3391d639c.png)
![unknown](https://user-images.githubusercontent.com/1374803/61865085-e7648300-aed2-11e9-8c8d-d2886c354b8e.png)

**Steps to reproduce**
There currently isn't a know way to reproduce. It happens randomly after some time. I've spent about 40 hours on this map.

**Build version**
0.74 2019-07-19 itch

**Platform**
Windows 10 Pro 1903 (Build 18362.239)

**Reproducibility**
Rare.

**Save file**
[https://cdn.discordapp.com/attachments/426912534428254219/603891486928666654/riugjx35c8m.zip](https://cdn.discordapp.com/attachments/426912534428254219/603891486928666654/riugjx35c8m.zip)

**Log file**
[Player-prev.log](https://github.com/voxeltycoon/bug-tracker/files/3430964/Player-prev.log)

**Comment by kristofeI** [comment]
I can confirm, it happens to me sometimes, at some time some stores just stop using their storage.

**Comment by johnfb** [comment]
I believe I have encountered the same issue.

Screen shot of demand, freight station, and vehicle in question.
![image](https://user-images.githubusercontent.com/2014627/78803175-48aa8480-7984-11ea-982f-802f54d2783f.png)
If you load the save, what you will see is that the vehicle that is to deliver iron ore will stop and then continue full. No delivery will be made to the demand, it will continue to show 0 delivered in last 30 days.

My save at the at the time of that screen shot. https://cdn.discordapp.com/attachments/378964810861117451/697468312417730620/save.zip

I encountered this running version 0.80.1

Additionally, I recall destroying a train station that originally supplied that demand with iron ore not too long before this happened. So, that might be one avenue of exploration.

Even more details, I looked around in that save and noticed a couple other demands also doing the same thing. I then let it run until the demands leveled down. The amount delivered went negative when that happened. But they did seem to start accepting deliveries and then seem to then get things sorted out.

Further note, continuing my save on for a while, it seems to keep happening with others as well.

Even further note (much later) in the save linked here, it happens a lot and you only need to load it up and set going fast and it seems to happen quite frequently.

Update on my journey of discovery (2020-04-14): Having run the game for hours trying to reproduce this, I seem to have stumbled on something very very odd. It appears as though this problem will not happen if the speed is set at 2 or 3. It only happens if you run on the slowest speed. I want to do another test to see if I can make a procedure to reproduce this.

I hope this helps.
